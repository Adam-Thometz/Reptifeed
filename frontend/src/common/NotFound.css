.NotFound {
  width: 50vw;
  margin: 0 auto;
  border: 1px solid #D5CE3B;
  color: #D5CE3B;
  padding: 5%;
  background-color: rgb(70, 70, 70);
  border-radius: 25px;
}

.NotFound a {
  display: flex;
  justify-content: space-evenly;
  color: #D5CE3B;
  text-decoration: none;
}