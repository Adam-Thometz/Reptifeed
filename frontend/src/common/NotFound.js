import React from "react";
import { Link } from "react-router-dom";

const NotFound = () => {
  return (
    <div>
      <h4>The page you're looking for wasn't found</h4>
      <Link to="/">Go back home</Link>
    </div>
  );
};

export default NotFound;