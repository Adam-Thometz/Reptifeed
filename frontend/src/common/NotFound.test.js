import React from "react";
import { render } from '@testing-library/react'
import NotFound from "./NotFound";
import { MemoryRouter } from "react-router-dom";

test('it renders without crashing', () => {
  render(<MemoryRouter>
    <NotFound />
  </MemoryRouter>);
});

test('it matches snapshot', () => {
  const { asFragment } = render(
    <MemoryRouter>
      <NotFound />
    </MemoryRouter>);
  expect(asFragment()).toMatchSnapshot();
})