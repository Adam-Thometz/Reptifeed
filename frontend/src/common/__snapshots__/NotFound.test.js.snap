// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <h4>
      The page you're looking for wasn't found
    </h4>
    <a
      href="/"
    >
      Go back home
    </a>
  </div>
</DocumentFragment>
`;
